modules/packages/Channel.chpl

```diff
@@ -180,7 +180,7 @@ module Channel {
     /*
      Initialize a channel
 
-     :arg elt: The element type used for sending and receiving
+     :arg eltType: The element type used for sending and receiving
      :type elt: `type`
```

Comment:
I think it would make sense to remove this line (good catch on the argument name for the previous one!)